base/sort.jl

```diff
@@ -434,8 +434,8 @@ struct PartialQuickSort{L<:Union{Integer,Missing}, H<:Union{Integer,Missing}} <:
     lo::L
     hi::H
 end
-PartialQuickSort(k::Integer) = PartialQuickSort(missing, k)
 PartialQuickSort(k::OrdinalRange) = PartialQuickSort(first(k), last(k))
+@deprecate PartialQuickSort(k::Integer) PartialQuickSort(missing, k)
```

Comment:
Haven't use `@deprecate`, but it seems it should be placed into `base/deprecated.jl`.